src/main/java/core/basesyntax/service/FigureBehaviourService.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface FigureBehaviourService {

```

Comment:
bad name for the interface, I would suggest something like `Drawable`.

The term "behaviour" too generic, in our case we could find more specific word.

The term "Service" is commonly used in the context of a design pattern called the "Service Layer." It generally represents a layer that provides some business logic or functionality to the application.